_includes/nav.html

```diff
@@ -1,5 +1,5 @@
-<div class="row justify-content-between">
-  <div class="col-auto mb-2">
+<div class="row m-0 justify-content-between">
+  <div class="order-1 order-md-1">
```

Comment:
What we're doing is dynamically changing the order of elements based on screen size.

Reply:
that is a neat trick! are order-1 and order-md-1 bootstrap styles?